### ISSUE TITLE
The prose around "simulcast envelope" falsely implies that simulcast encodings can never be removed

### ISSUE THREAD

**Issue by docfaraday** [w3c/webrtc-pc#2723]
Spec says "Once the envelope is determined, layers cannot be removed.", but the language for sRD(answer) says that if rids are rejected by an answer, they are removed.

There are a couple of ways to fix this:

1. We remove this assurance from the section on "simulcast envelope", or
2. We only allow the first answer to remove rids from [[SendEncodings]].

Disallowing an answer to remove rids on a previously negotiated sender is probably not appropriate, since this would violate the simulcast spec, which requires the offerer to handle this case regardless of whether this is the initial negotiation or not. I think option 1 is the correct course of action here.

**Comment by aboba** [comment]
Re-reading Section 5.4.1, I agree that there is an issue. While the section seems to imply that the simulcast envelope is established by `addTransceiver()` alone, this is not correct, if only because the answer can reject layers/RIDs.  My inclination is that the envelope is only really determined once the initial negotiation has completed.  However, this does raise the question of whether `setParameters()` can modify parameters originally set by `addTransceiver()` prior to completion of the initial negotiation.

**Comment by docfaraday** [comment]
I should also note that if we do allow rid reordering in a reoffer, we should not mess with [[SendEncodings]] in any way as a result. We should probably just create our answer using the same order, and let it have no other effect.

**Comment by youennf** [comment]
> we should not mess with [[SendEncodings]]

I agree with not changing [[SendEncodings]], but then we have descriptions that are not synchronised with [[SendEncodings]].
It seems rejecting in that case is something we could do and would avoid corner cases as well as potentially simplify implementations.
I would think we already sometimes reject descriptions that are valid SDPs so I am not sure what we gain in not rejecting?

**Comment by docfaraday** [comment]
Generally, my preference is to follow the SDP negotiation rules in the IETF specifications wherever possible. While throwing errors when we encounter something valid-but-weird is one way to simplify the web side of things, from an IETF perspective this is a bug in our implementation of RFC 8853.

**Comment by aboba** [comment]
Feedback from June 19, 2022 WEBRTC WG Virtual Interim: "Ready for PR"

**Comment by jan-ivar** [comment]
> Spec says "Once the envelope is determined, layers cannot be removed.",

This was added in https://github.com/w3c/webrtc-pc/pull/2081 and, as I read it, is only talking about envelopes established by remote offers to receive simulcast, and the limitations on browser client answers to those. In that case the statement seems true, because SLD disallows anything but unadulterated createAnswer, and AFAIK no API allows JS to reduce layers in answers.

I've tried to clarify this in https://github.com/w3c/webrtc-pc/pull/2760.
